src/components/ParsedBotMessageBody.tsx

```diff
@@ -54,7 +54,9 @@ type MetaData = {
 export default function ParsedBotMessageBody(props: Props) {
   const { message, tokens } = props;
   const { enableSourceMessage } = useConstantState();
-  const data_ = (message as UserMessage).data as string;
+  const data_ =
+    (message as UserMessage).data === '' ? '{}' : (message.data as string);
```

Comment:
Can message or message.data be null or undefined?

Reply:
This is the type of `message.data` (defined in the chat sdk). I thought this would be just undefined or a valid string(not 0 length of string), but I found it sometimes just goes a empty string `''`. 

```
data?: string;
```